app/pages/login/login.js

```diff
@@ -70,9 +72,19 @@ export let Login = translate()(React.createClass({
   },
 
   render: function() {
+    const { t } = this.props;
     var form = this.renderForm();
     var inviteIntro = this.renderInviteIntroduction();
 
+    var cookieText = (
+      <Trans i18nKey="html.cookie-content">
+        <div>
+          Please consult our <a href="https://s3-eu-west-1.amazonaws.com/com.diabeloop.public-assets/data-privacy.pdf" className="link-cookieConsent" target="_blank">Data Privacy</a> and our <a href="https://s3-eu-west-1.amazonaws.com/com.diabeloop.public-assets/terms.pdf" className="link-cookieConsent" target="_blank">Terms of Use</a>
```

Comment:
Add to the <a> elements: rel='noopener noreferrer'
See: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md